export-ingester/export_ingester/main.py

```diff
@@ -1,16 +1,22 @@
 import asyncio
-from datetime import date
+from datetime import date, timedelta
 
 from .api_client import ExportParams
 from .config import Settings
 from .ingest import get_managed_export_ingester
+from .utils import get_range
 
 
 async def main():
     settings = Settings()
-    remote = f"daily-uploads/{date.today()}.json"
+    remote = f"/scratch-shared/amftrack2024/daily/{date.today()}.json"
+    time_range = 4
+    # TODO these two need to become either parameter or environment variables
+    date_ = date.today() - timedelta(days=time_range)
```

Comment:
is this correct? I think you're pulling data between 4 and 8 days ago. Is that what you want?

Reply:
Oh then, I did get confused. I want everything between today and 4 days before. Fixed